### PR TITLE
fix: [HCPSDKFIORIUIKIT-2043] resolve UIFont instance from Swift pass to SwiftUI can't be resized and empty text disable accessibility.

### PR DESCRIPTION
**Pull request by AllenCao001**

### CODE REVIEW

File: Sources/FioriSwiftUICore/DataTable/LayoutData.swift [hunk 1]
```diff
@@ -143,7 +143,13 @@ class LayoutData {
                 let title = item.text
                 var uifont: UIFont
                 if let tmpUIFont = item.uifont {
-                    uifont = tmpUIFont
+                    // `item.uifont` is passed by developer, although is a preferred font but can't resize according to system in SwiftUI. So need to redefine the UIFont instance.
+                    if let styleName = tmpUIFont.fontDescriptor.fontAttributes[.textStyle] as? String {
```
Comment (billzhou0223) [inline Sources/FioriSwiftUICore/DataTable/LayoutData.swift:147]: In order for the text to be resizable, developes are responsible for passing in a scaled font. We, as SDK, are not supposed to force this conversion.

**Review by billzhou0223 — CHANGES_REQUESTED** [review]
@shengxu7

**Review by shengxu7 — APPROVED** [review]

File: Sources/FioriSwiftUICore/DataTable/LayoutData.swift (hunk 1 above)
Comment (AllenCao001) [reply Sources/FioriSwiftUICore/DataTable/LayoutData.swift:147]: At now, the issue is developers had init a `UIFont` instance via `UIFont.preferredFont()` in SDK and passed it to SwiftUI, but the text doesn't be resizable in SwiftUI. In order to fix the issue, so need to redefine `UIFont` in SwiftUI, it is as the above code.

**Review by billzhou0223 — APPROVED** [review]

**Comment by AllenCao001** [comment]
When it can be merged into the main branch? Whether this pull request has any question? Because I need to fix the issue-2043.

**Pull request merged**